### PR TITLE
Fix: broken code snippets in bitesize-clientdriven.md

### PR DESCRIPTION
**Pull request by Elfi0Kuhndorf**
fixed broken script references and one typo

#### **WARNING**: Please make your PR against the `develop` branch.
- - -
**Select the type of change:**

- [x] Small Changes - Typos, formatting, slight revisions
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**

Fix broken code snippets and a typo mentioned in https://docs.google.com/document/d/1b0cdh-buY_GtWrpRBoNt32cEpS37i2Pmeh5dMjMuj0Q/edit#heading=h.dahhpvq8vb5s

**Issue Number:**

[MTT-8542](https://jira.unity3d.com/browse/MTT-8542)

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=1267) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=1267) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=1267) it.</sub>

File: docs/learn/bitesize/bitesize-clientdriven.md [hunk 1]
```diff
@@ -17,10 +17,10 @@ Making movements feel responsive while staying consistent over multiple game exe
...
+It uses the ClientNetworkTransform component and moves your own player's position client side, [client authoritatively](../dealing-with-latency.md#allow-low-impact-client-authority). Movement is leveraged through the use of Unity's Starter Assets, the [Third Person Character Controller](https://assetstore.unity.com/packages/essentials/starter-assets-third-person-character-controller-196526).
 
 ```csharp reference
-https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/blob/v1.2.1/Basic/ClientDriven/Assets/StarterAssets/ThirdPersonController/Scripts/ThirdPersonController.cs#L274-L276
+https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/blob/v1.2.1/Basic/ClientDriven/Assets/StarterAssets/ThirdPersonController/Scripts/ThirdPersonController.cs#L155-L162
```
Comment (Elfi0Kuhndorf) [inline docs/learn/bitesize/bitesize-clientdriven.md:23]: @fernando-cortez  How can I preview how it actually looks on the page? When I preview it I always only see the line " https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/blob/v1.2.1/Basic/ClientDriven/Assets/StarterAssets/ThirdPersonController/Scripts/ThirdPersonController.cs#L155-L162"

Comment (fernando-cortez) [reply docs/learn/bitesize/bitesize-clientdriven.md:23]: @Vic-Cooper we used to have preview deployments. I'm seeing that the last few to be generated were around May 2023.

Is there a way to preview changes nowadays?

Comment (jabbacakes) [reply docs/learn/bitesize/bitesize-clientdriven.md:23]: You can build things locally using Docusaurus, which is what we've been using as a local/in-development test. And then there's a staging build when things are merged into the `develop` branch.

**Review by fernando-cortez — APPROVED** [review]

**Pull request merged**